src/OpenTelemetry.Extensions/Internal/OpenTelemetryExtensionsEventSource.cs

```diff
@@ -42,4 +42,19 @@ public void LogProcessorException(string @event, string exception)
     {
         this.WriteEvent(1, @event, exception);
     }
+
+    [NonEvent]
+    public void LogRecordFilterException(string? categoryName, Exception ex)
+    {
+        if (this.IsEnabled(EventLevel.Warning, (EventKeywords)(-1)))
+        {
+            this.LogRecordFilterException(categoryName, ex.ToInvariantString());
+        }
+    }
+
+    [Event(2, Message = "Filter threw an exception, request will not be collected. CategoryName: '{0}', Exception: {1}.", Level = EventLevel.Warning)]
```

Comment:
please modify the message to reflect the actual log.

Reply:
Sorry, I do not understand what you mean. Can you be more specific?

Reply:
**"request will not be collected"** -- this message is not relevant here. Maybe copy-paste error?

Reply:
Ok, maybe "log record will not be attached" will be better?

Reply:
^ is good. Equally important is to state what is unaffected : the logs records would flow to its pipeline unaffected.